qubes.SaltLinuxVM

```diff
@@ -51,4 +51,8 @@ $target_vm:
 EOF
 export PATH="/usr/lib/qubes-vm-connector/ssh-wrapper:$PATH"
 
-salt-ssh -w "$target_vm" $salt_command
+# Ensure the arguments for salt-ssh are tokenized correctly.
+# Without this parsing, the salt_command is just split on whitespace.
+# E.g.: test.echo 'foo bar' -> ['test.echo', "'foo", "bar'"]
+readarray -t -d '' salt_args < <(xargs printf '%s\0' <<<"$salt_command")
+salt-ssh -w "$target_vm" "${salt_args[@]}"
```

Comment:
What about using `eval` here instead?
```suggestion
# Ensure the arguments for salt-ssh are tokenized correctly.
# Without this parsing, the salt_command is just split on whitespace.
# E.g.: test.echo 'foo bar' -> ['test.echo', "'foo", "bar'"]
eval "salt-ssh -w $target_vm $salt_command"
```

Reply:
Well, that's certainly less complex and does the job. I almost never use `eval`, so I wasn't certain this wouldn't lead to surprising side effects in some edge cases and discarded it early.

The code I proposed should keep the semantics of `salt_command` being an array of parameters intact, but relies on a non-ancient bash version.

I'm fine with either if you say that's fine, so if you prefer `eval`, I'm happy to adapt it.

Reply:
Input comes from a trusted Python script and is quoted using `shlex.quote()`, which escapes things for the shell, so `eval` is the simplest approach.

Reply:
I have committed your suggestion now. Qube names are not escaped, but (as expected) are limited to safe characters. Thanks for your quick response :)

Reply:
You’re welcome!